knowhere/index/vector_index/gpu/IndexIVFSQHybrid.cpp

```diff
@@ -9,13 +9,15 @@
 // is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 // or implied. See the License for the specific language governing permissions and limitations under the License
 
+#ifdef KNOWHERE_GPU_VERSION
```

Comment:
need not add "KNOWHERE_GPU_VERSION" in this file, the whole file should not be compiled in CPU mode.